.gitmodules

```diff
@@ -47,12 +47,13 @@
 [submodule "third_party/robin-map"]
 	path = third_party/robin-map
 	url = https://github.com/Tessil/robin-map.git
-[submodule "third_party/sql-parser"]
-	path = third_party/sql-parser
-	url = https://github.com/hyrise/sql-parser.git
 [submodule "third_party/tpcds-kit"]
 	path = third_party/tpcds-kit
 	url = https://github.com/hyrise-mp/tpcds-kit.git
 [submodule "third_party/zstd"]
 	path = third_party/zstd
 	url = https://github.com/facebook/zstd.git
+[submodule "third_party/sql-parser"]
```

Comment:
Why the change? I think the module have been sorted alphabetically before.

Reply:
I only changed the submodules using GIT CLI to include the parser's feature branch with the `BIGINT` type, the submodules will be reverted once this feature branch is merged into the parser's master.